packages/fluentui/docs/src/components/DocsLayout.tsx

```diff
@@ -100,7 +100,7 @@ class DocsLayout extends React.Component<any, any> {
               {
                 // adjust Teams' theme to Semantic UI's font size scheme
                 componentVariables: {
-                  HierarchicalTreeItem: {
+                  TreeItem: {
```

Comment:
Can you please check this one? I don't think that it really works as there is no `componentVariables`

Reply:
True story, removed